exercises/robot-name/.meta/exercise-data.yaml

```diff
@@ -0,0 +1,63 @@
+exercise: RobotName
+package_comment: "# This will be class 'RobotName', defined using Moo (https://perlmaven.com/oop-with-moo)"
+lib_comment: '# Find modules in the same dir as this file.'
+plan_comment: '# This is how many tests we expect to run.'
+
+moo: true
+methods: new name reset_name
+
+plan: 7
+  # plan includes can_ok of `methods` plus the tests below.
+
+# Tests: inline here, since there is no canonical-data.json for this exercise
+tests: |-
+  my $robot = RobotName->new;
+  isa_ok $robot, 'RobotName';
+
+  my $name  = $robot->name;
+  ok $robot->name =~ /[A-Z]{2}[0-9]{3}/, 'Name should match schema';
```

Comment:
Although this regex was in the existing test I'd say it'd be worth adding `^` and `$` to it here.